go-for-eat/localStorage.js

```diff
@@ -12,10 +12,20 @@ export const loadState = () => {
 export const saveState = (state) => {
   try {
     const serializedState = JSON.stringify({
-      authentication: state.authentication
+      authentication: state.authentication,
     });
     Expo.SecureStore.setItemAsync('state', serializedState);
   } catch (err) {
     console.log(err);
   }
 };
+
+export const logoutStorage = () => {
+  try {
+    const serializedState = JSON.stringify({});
+    Expo.SecureStore.setItemAsync('state', serializedState);
```

Comment:
If you return this, you'll make logoutStorage to be an asynchronous operation.
And the try catch in the level above, so next to the business logic.